src/screens/SignedChatScreen/index.tsx

```diff
@@ -28,8 +28,8 @@ const SignedChatScreen = () => {
     goToChatInfoScreen,
     sendChat,
     updateChatContinuity,
-    flatListRef: scrollRef,
-    scrollContext
+    scrollContext,
+    flatListRef: scrollRef
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The order of destructuring properties from `useChatScreenHook` has been changed. This is not an issue as the order of destructuring does not affect functionality, but it's important to maintain consistency in code style across the project.



<!-- This is an auto-generated comment by OSS CodeRabbit -->